### DISCUSSION TITLE
Tai可以尝试添加时间轴功能吗？

### DISCUSSION THREAD

**Discussion by RGthx** [Planshit/Tai#106] [Ideas]
Tai可以尝试添加时间轴功能吗？
就是新建一个栏目（视图）
可以直观展示从哪个时刻到哪个时刻，电脑使用的程序
同时也可以沿用分类里的颜色标记来标记轴段
类似于微软的时间轴

**Comment by noberumotto** [comment]
能提供点截图或者应用吗

**Reply by RGthx** [reply]
请原谅我的图很丑，以及我回复的很晚，谢谢
![image](https://user-images.githubusercontent.com/118524197/212820946-c68be52a-fab5-4089-9bcd-8fa04d9cecb9.png)
这算是一种思路，我希望能把用户的一天的时间线以这种方式呈现出来
线条的颜色与分类中设置的相同
空闲（未使用的时间）统一使用更细的黑线（或白线）
而下面以文字的方式具体记录从哪时哪刻到哪时哪刻使用的软件
可以尝试把这个时间轴扩展到每周，但我认为这可能效果并不好

同时也可以加入”时间小格“功能（类似于”人生小格“（参考https://tools.miku.ac/lifecount/））
不过我们选择也是以一天为总时长，1min或1s为基础格子单位，对应格子表示了对应时间
（例如第36个格子表示0：35-0：36）
但以一秒为时间单位的话格子数过多，而以一分钟为时间单位的话不足一分钟的使用时间难以表示
（例如我可能用十秒上了个b站，关了，又看了眼qq 又关了，显然，不足一分钟，难以表示）

大概就是这种思路吧，我更倾向于第一种，不过时间轴也不非得是我画的轴状，事实上，可以让他更简洁，更好看

**Reply by yongpz** [reply]
这个功能感觉很nice

**Reply by CGH20171006** [reply]
我也想提出这个建议，相关的应用当然是rescueTime或者manicTime啦，现在就在用manictime，时间轴功能更完整，读取的信息更详细，但是图形界面丑陋哈哈

**Reply by Stargazed-Dreamer** [reply]
我也想提出这个建议来着

**Reply by majianchuan** [reply]
我开发的支持时间轴功能，欢迎尝试：https://github.com/majianchuan/ScreenTimeTracker
